examples/aws/terraform/ha-autoscale-cluster/output.tf

```diff
@@ -0,0 +1,35 @@
+//Output providing the bastion public IP, auto scaling group names and how 
+//to get the private IPs to jumphost through the bastion. 
+
+
+
+output "z_bastion_instance_public_ip" {
+  value = {
+  for instance in aws_instance.bastion:
+  instance.id => format("ip %s %s%s %s",instance.public_ip, ", Add identity file via ssh agent (ssh-add identityfile.pem)\n and you can jump host to servers. \nEx:\nssh -J ec2-user@", instance.public_ip, " ec2-user@<proxy/auth/node ip>") 
+}
+  description = "AWS Bastion EC2 IP. Add AWS identity file to your ssh agent and you can jump host through the bastion.  ssh -J <bastion-ip> <proxy/auth ip>"
```

Comment:
I don't think we should be providing detailed instructions via command line or outputs, we should do all of this via our docs. I'm cool with the IP of everything to make it easier to connect straight away, but I don't want to try and teach people how to use Terraform or AWS via the CLI.

I'm hoping that once the Teleport Terraform provider is up and running, we can skip all this need to SSH to the bastion anyway and just let someone `tsh login` to their cluster straight away.